src/avram/tasks/db/verify_connection.cr

```diff
@@ -21,6 +21,7 @@ class Db::VerifyConnection < LuckyCli::Task
 
         ▸ Make sure Postgres is running.
         ▸ Check connection settings in config/database.cr.
+        ▸ If the database has not been created yet, run 'lucky db.create'
```

Comment:
How does that look @jwoertink?

Reply:
ah yup! I think that reads a lot more clear. Thanks!